core/app/models/spree/tax/application.rb

```diff
@@ -0,0 +1,51 @@
+module Spree
+  module Tax
+    class Application
```

Comment:
I think we need a different class name here. `Application` has too strong another meaning